src/super_gradients/training/transforms/transforms.py

```diff
@@ -410,7 +410,7 @@ def __call__(self, sample: Union[dict, list]):
             all_samples = [sample] + sample["additional_samples"]
 
             for i_mosaic, mosaic_sample in enumerate(all_samples):
-                img, _labels = mosaic_sample["image"], mosaic_sample["target"]
+                img, _labels, = mosaic_sample["image"], mosaic_sample["target"]
```

Comment:
I think this was already fixed when @Louis-Dupont fixed lint

Reply:
fixed https://github.com/Deci-AI/super-gradients/pull/294/commits/8bdf12438e74d3efb245b92baa1ff9ffef3c7306